lib/python/ray/worker.py

```diff
@@ -1175,6 +1226,25 @@ def register_class_for_serialization(worker_info):
     serialization.add_class_to_whitelist(cls, pickle=pickle)
   worker.run_function_on_all_workers(register_class_for_serialization)
 
+def log(event_type=None, contents=None, worker=global_worker):
+  """Log an event to the global state store.
+
+  This adds the event to a buffer of events locally. The buffer can be flushed
+  and written to the global state store by calling flush_log().
+
+  Args:
+    event_type (str): The type of the event.
+    contents: More general data to store with the event.
```

Comment:
let's demand here that content is a little more restricted, say dicts with only strings as keys and vals, to make it trivially json serializable